misc/dockertest.py

```diff
@@ -271,6 +294,24 @@ def notary_list(fout, repo):
     return [line.strip().split() for line in lines[2:]]
 
 
+def test_build(fout, image, docker_version):
+    """
+    Build from a simple Dockerfile and ensure it works with DCT enabled
+    """
+    # build
+    # simple dockerfile to test building with trust
+    dockerfile = "FROM {0}:{1}\nRUN sh\n".format(image, docker_version)
```

Comment:
Should this call `clear_tuf()` first?  When we were testing manually, this succeeded if the TUF data existed because it fell back on the cache, even if there was a problem with the server.  

I'm less sure about whether we care about removing the tag first, since I think trusted builds will probably check the trust server anyway even if the image exists locally - I forget if that impacted anything at all when we were testing yesterday.